### ISSUE TITLE
Improve examples in the documentation

### ISSUE THREAD

**Issue by gazpachu** [googlemaps/react-wrapper#114]
Thanks for this package! I was wondering if it'd be possible to expand the documentation a bit more.

In example, what's `<MyMapComponent />` meant to be? How can we pass options to the Google Maps API?

At the moment, after following the example in the documentation, there's no map rendered.

**Comment by apraMuh** [comment]
lack of documents agreed

**Comment by jpoehnelt** [comment]
This library does not provide any specific map component, just a parent component that can be used to wrapped child components that use `google.maps`.

**Comment by joshualan** [comment]
I agree, any full simple working example would be great. Literally give an example of what MyMapComponent could contain.

**Comment by karlbalagtey** [comment]
yeah I agree with the rest, an example would be great
